data/json/items/tool/entry_tools.json

```diff
@@ -142,6 +142,7 @@
     "qualities": [ [ "HAMMER", 1 ], [ "PRY", 3 ], [ "COOK", 1 ] ],
     "use_action": [ "CROWBAR" ],
     "melee_damage": { "bash": 45 },
+    "weapon_category": [ "BATONS" ],
```

Comment:
Nothing sixty inches long qualifies as a baton.  Specifically anything that sheath's as a spear or requires always_twohand should be in the baton category.
```suggestion
```